### PR TITLE
FEAT : Video card with detailing 

### PR DESCRIPTION
**Pull request by t007rushi**
## Explore Page created
 - with video card Image
 - with detailing
 
 Home Page => navigate Expolre 
 
 recording

### CODE REVIEW

File: src/frontend/Layout/Header/header.css [hunk 1]
```diff
@@ -10,6 +10,9 @@ text-decoration: none;
 .header-bar {
   background-color: var(--accent--one-color);
   color: var(--foreground-color);
+  z-index:999;
```
Comment (vijaytembugade) [inline src/frontend/Layout/Header/header.css]: be specific about giving z index. May cause issues later.

File: src/frontend/components/Browse/browse.css [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  width: 100%;
+  top:4.3rem;
+  align-items: flex-start;
+}
+
+.options-mob-view{
+    padding: 10px 0;
+    margin-left: 1rem;
+}
+/* rgba(0,0,0,.6) */
```
Comment (vijaytembugade) [inline src/frontend/components/Browse/browse.css]: remove comments

File: src/frontend/components/Row/Row.jsx [hunk 3]
```diff
@@ -16,10 +20,10 @@ export const Row = ({ cat, title }) => {
   const Games = videos.filter((vid) => vid.categoryName === cat);
   const ArrowHandler = (dir) => {
     if (dir === "right") {
-      setVal(v => v - 20);
+      setVal((v) => v - 20);
```
Comment (vijaytembugade) [inline src/frontend/components/Row/Row.jsx]: try to give proper variable names!
eg. `prevValue` or `prevState`

**Review by vijaytembugade — APPROVED** [review]
Looks good. I like how you optimise the NavLink option by giving it in array of object.
Good to move ahead
image

File: src/backend/db/videos.js [hunk 4]
```diff
@@ -7,235 +7,296 @@ import { v4 as uuid } from "uuid";
 
 export const videos = [
   {
-    _id: uuid(),
-    title: "god of war",
+    _id: "-IJuKT1mHO8",
+    title: "far cry 6",
```
Comment (horsemaker) [inline src/backend/db/videos.js]: Just a knick bit. Maybe capitalize the first letter of every word.

File: src/backend/db/videos.js [hunk 5]
```diff
@@ -7,235 +7,296 @@ import { v4 as uuid } from "uuid";
...
     image:
       "https://ik.imagekit.io/t007rushi/VideoCardImages/pc/farcry6_KgK7f2lbZE.png?ik-sdk-version=javascript-1.4.3&updatedAt=1649669798331",
```
Comment (horsemaker) [inline src/backend/db/videos.js:13]: Maybe you could make a util function to generate this link as you have to keep repeating stuff here.

File: src/frontend/Layout/Header/header.css (hunk 1 above)
Comment (horsemaker) [reply src/frontend/Layout/Header/header.css]: Rightly pointed out by @vijaytembugade!

File: src/frontend/components/Row/Row.jsx (hunk 3 above)
Comment (horsemaker) [reply src/frontend/components/Row/Row.jsx]: Again, rightly pointed out by @vijaytembugade!

**Review by horsemaker — APPROVED** [review]
Looks good! 👌

**Pull request merged**